zilliqa/Cargo.toml

```diff
@@ -13,6 +13,10 @@ path = "src/lib.rs"
 name = "zilliqa"
 path = "src/bin/zilliqa.rs"
 
+[[bin]]
+name = "validator_oracle"
+path = "src/bin/validator_oracle.rs"
```

Comment:
It will interact with contracts deployed on the blockchain, but I suspect this isn't what you mean? It does rely heavily on code in the crate. I can look into moving this a level up, sure.